docs/modules/Db.md

```diff
@@ -60,6 +60,18 @@ if you run into problems loading dumps and cleaning databases.
              ssl_ca: '/path/to/ca-cert.pem'
              ssl_verify_server_cert: false
              ssl_cipher: 'AES256-SHA'
+             
+## Example 2
```

Comment:
Ok, please move this example to `Codeception\Module\Db.php` into the docblock. This file is generated from it